printable-recipe-generator/src/assets/recipe.ejs

```diff
@@ -266,6 +280,17 @@
         <% } %>
     </div>
 </footer>
+
+<%if (qrImageDataUrl && qrImageDataUrl!=null || qrImageDataUrl?.length>0) { %>>
+
+<div class="qr-section">
+    <p>📱 Open in App</p>
+    <img src="<%= qrImageDataUrl %>" alt="Scan to open app" />
+    <p>feastbraze://recipe/<%= recipe.id %></p>
```

Comment:
Is this necessary? I can't imagine a user ever typing it in. Maybe one for Murray

Reply:
Yes, confirmed it with Murray. According to him it will be simple an Image, a QR code with nothing extra details on it,so will remove it.